packages/amplify-category-function/src/provider-utils/awscloudformation/utils/layerParams.ts

```diff
@@ -1,12 +1,15 @@
 import { FunctionRuntime, ProviderContext } from 'amplify-function-plugin-interface';
+import _ from 'lodash';
+import path from 'path';
 
 export type LayerParameters = {
   layerName: string;
-  layerPermissions: Permissions;
+  layerPermissions: Permissions[];
```

Comment:
why do we have both layerPermissions and layerVersionMap? Doesn't the layerVersionMap contain the permissions for each version (including the current one)?